plugin/src/utils/cache.ts

```diff
@@ -337,7 +343,7 @@ export const restoreHardCachedNodes = async ({
       }
 
       // restore each node
-      await helpers.actions.createNode(remoteNode)
+      helpers.actions.createNode(remoteNode)
```

Comment:
I have a theory that, depending on how we are transpiling async/await, that this would introduce a performance improvement. typescript revealed this was an unnecessary use of await

Reply:
nice!

Reply:
I think the core type on this is probably wrong 🤔 or TS is wrong somehow. If you check the return type in the actions docs it returns a promise which resolves when all cascading createNode calls are finished https://www.gatsbyjs.com/docs/actions/#createNode

Reply:
ok! i can just tsignore it for now. it doesn't really impact any inference outside itself